frappe/core/doctype/user/user.py

```diff
@@ -240,9 +240,9 @@ def send_welcome_mail_to_user(self):
 
 		link = self.reset_password()
 
-		app_title = [t for t in frappe.get_hooks('app_title') if t != 'Frappe Framework']
+		app_title = [t for t in frappe.get_hooks('app_title') if t == 'ERPNext']
```

Comment:
@britlog, you will need to call the method first, `frappe.get_attr(method_name)` will return the object of method

e.g `get_site_info = frappe.get_attr(method_name)()`